Product/Production/Adapters/General/CONNECTAdminGUI/src/main/webapp/viewCertDialog.xhtml

```diff
@@ -123,7 +124,17 @@
                      <div class="form-group col-sm-12">
                         <h:outputText> Note: "-" denotes empty value </h:outputText>
                      </div>
+                     
+                       <div class="form-group">
+                        <div class="col-sm-12">
+                            <div class="form-button-row">
+                                <p:commandButton  id="update" value="UPDATE CERTIFICATE" actionListener="#{certificateBean.updateSelectedCertificate}" update=":certTabView,:certTabView:trustStoreForm,:certTabView:keyStoreForm, viewCertDlgForm:viewCertErrorMsg" styleClass="pull-right"/>
```

Comment:
Do we have all caps button title on the page?

Reply:
Done